working/wildcards/feature-specification.md

```diff
@@ -317,37 +317,54 @@ class C {
 
 ### Super parameters
 
-An occurrence of `super._` as a declaration of a formal parameter in a
-constructor is a compile-time error. This error also occurs in the case
-where the super parameter has an explicitly declared type and/or default
-value.
+The positional formal parameter `super._` is still allowed in generative
+constructors.
 
-*`super._` is not an error everywhere: In a method body it could be an
-invocation of an inherited getter named `_`.*
+`super._` forwards the argument's value to the super constructor invocation.
 
 ```dart
-class B {
-  final _;
-  B(this._);
+class A {
+  final int _;
+  A(this._);
 }
 
-class C extends B {
-  C(super._); // Error.
+class B extends A {
+  B(super._); // OK.
 }
 ```
 
-*The desugared meaning of a super parameter includes a reference to the
-parameter in the initializer list of the enclosing constructor declaration,
-but such references are not possible when the parameter name is a
-wildcard.*
+Similar to `this._`, no parameter named `_` is bound. `_` can't be accessed in
+the initializer list.
 
-*It may seem inconvenient that `super._` "does not work" in the last
-example, but we do not wish to create exceptions about the ability to refer
-to a declaration whose name is a wildcard, just so we can support this
-particular usage. The advice would be to use a different name than `_` for
-the instance variable in `B` in the first place, or using a different name
-in `B` and possibly `// ignore` a lint that may warn about the names being
-different.*
+```dart
+class A {
+  final int _;
+  A(this._);
+}
```

Comment:
(Here the `_` is used, to show the "Prints inherited field". Could consider showing the difference between the parameter and the field more distinctly:
```dart
 class Capacity {
   final int _;
   Capacity(int value, int limit) : assert(value <= limit), _ = limit = value;
 }
  class B extends Capacity {
    B(super._, super.max)
        : assert(max >= 0),  // OK, `max` in scope.
          assert(_ >= 0) {  // Error, no `_` in scope.
      print(_); // prints value of `A._`
      print(max); // Error, no `max` in scope.
    }
  }
```